python/src/otel/otel_sdk/requirements.txt

```diff
@@ -2,5 +2,35 @@ opentelemetry-sdk==1.26.0
 opentelemetry-exporter-otlp-proto-http==1.26.0
 opentelemetry-distro==0.47b0
 opentelemetry-instrumentation==0.47b0
-opentelemetry-semantic-conventions == 0.47b0
+opentelemetry-semantic-conventions==0.47b0
 opentelemetry-propagator-aws-xray==1.0.2
+
+# Instrumentation dependencies
+opentelemetry-instrumentation-aiohttp-client==0.47b0
```

Comment:
How much does moving all this to the lambda layer increase the layer size? Is that enough to be concerned about for adding to the cold start time?

Reply:
I simulated locally to check what is the delta:

without installing all dependencies
```
python3 -m pip install -r otel_sdk/requirements.txt -t ./deps && \
  python3 -m pip install -r otel_sdk/nodeps-requirements.txt -t ./deps --no-deps
du -h --max-depth 0 deps
25M     deps
```

installing all transitive dependencies
```
python3 -m pip install -r otel_sdk/requirements.txt -t ./deps && \
  python3 -m pip install -r otel_sdk/nodeps-requirements.txt -t ./deps --no-deps
du -h --max-depth 0 deps
27M     deps
```

The impact on the cold startup will be negligible.

Reply:
Looks like the [last version](https://github.com/open-telemetry/opentelemetry-lambda/actions/runs/9995251242) was 7.4 MB.  We can see where things end up with the next release.

Reply:
So an added 1mb.... that's not insignificant. 🤷